api/proto/teleport/plugins/v1/plugin_service.proto

```diff
@@ -136,4 +148,8 @@ service PluginService {
   // GetAvailablePluginTypes returns the types of plugins
   // that the auth server supports onboarding.
   rpc GetAvailablePluginTypes(GetAvailablePluginTypesRequest) returns (GetAvailablePluginTypesResponse);
+
+  // GetPluginStaticCredentials returns static credentials that are requested. Only accessible by RoleAdmin and,
+  // in the case of Teleport Assist, RoleProxy.
+  rpc GetPluginStaticCredentials(GetPluginStaticCredentialsRequest) returns (GetPluginStaticCredentialsResponse);
```

Comment:
I would expect a Get method to return a single element, but this returns many. Should we consider a rename to a less-usual verb like "Search"?

* https://cloud.google.com/apis/design/standard_methods#get
* https://cloud.google.com/apis/design/custom_methods#common_custom_methods

Reply:
I'm fine with Search, that makes sense.